packages/components/src/text-control/index.styles.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
We have some file naming inconsistencies with the `.js` containing CSS-in-JS styles.
I don't think it's anyone's fault :).

https://github.com/WordPress/gutenberg/tree/master/packages/components/src/card
https://github.com/WordPress/gutenberg/tree/master/packages/components/src/text

Would be good to consolidate moving forward 🙌.
If I recall... I know we'd prefer to avoid `.` notation in file names.

I had originally approached it by containing things in a `styles` directory to contain localized style utils, mixins, or styles from sub-components. I'm open to modifying.

Would love to hear your thoughts!